src/mscorlib/shared/System/TimeZoneInfo.Win32.cs

```diff
@@ -11,6 +11,20 @@
 using System.Threading;
 using Microsoft.Win32;
 
+#if CORERT
```

Comment:
Right, there is small adjustment to make it fit into https://github.com/dotnet/corert/tree/master/src/System.Private.CoreLib/shared